### PR TITLE
Add citation information

### PR DESCRIPTION
**Pull request by riedgar-ms**
Provide a BibTeX entry for the whitepaper which describes the Fairlearn package. As we shift to a more open governance model, this should be updated with a more general publication.

Inspired by #641

### CODE REVIEW

File: docs/about/index.rst [hunk 1]
```diff
@@ -28,6 +28,25 @@ Governance
...
+        author = {Bird, Sarah and Dud{\'i}k, Miro and Edgar, Richard and Horn, Brandon and Lutz, Roman and Milan, Vanessa and Sameki, Mehrnoosh and Wallach, Hanna and Walker, Kathleen},
+        title = {Fairlearn: A toolkit for assessing and improving fairness in {AI}},
+        institution = {Microsoft},
+        year = {2020},
+        month = {May},
+        url = "https://www.microsoft.com/en-us/research/publication/fairlearn-a-toolkit-for-assessing-and-improving-fairness-in-ai/",
+        number = {MSR-TR-2020-32},
+    }
+
+As we shift to a more open governance model, we expect to update this citation.
```
Comment (romanlutz) [inline docs/about/index.rst:48]: I'm not sure I understand what this means. In my mind, it seems reasonable to write a follow-up at some point (I have no timeline in mind) when we expand the focus beyond binary classification & regression to more complex tasks, or perhaps when we manage to get a few examples worked out. That said, I am not aware of plans and this seems to be hinting at something, so I just want to clarify :-)

**Review by romanlutz — APPROVED** [review]

File: docs/about/index.rst (hunk 1 above)
Comment (MiroDudik) [reply docs/about/index.rst:48]: Agreed with @romanlutz. we will add a new citation if there's a new paper! and we should write a new paper when there's a good reason to write a new paper--so when we go neutral, we should reflect if there's enough for a new paper.

For now, let's just drop "As we shift ..."

**Review by MiroDudik — APPROVED** [review]
Good to go once "As we shift ..." sentence is removed.

File: docs/about/index.rst (hunk 1 above)
Comment (riedgar-ms) [reply docs/about/index.rst:48]: I had it in to minimise the appearance of excessive MS influence. I'll remove.

**Pull request merged**